config.h

```diff
@@ -13,20 +13,22 @@
  */
 
 // General settings
-#define MY_BAUD_RATE 9600
+#define MY_BAUD_RATE 115200
```

Comment:
Hi, it is a good idea but I was looking into something requiring no manual changes. Like compiling a config.h file if present, otherwise the config.h.sample (https://github.com/mysensors/NodeManager/issues/75). Because like it is you're right, it is a nightmare while debugging because you need to revert all the changes back once done. I still have no idea how to achieve it, btw :)